spec.html

```diff
@@ -27890,16 +27891,11 @@ <h1>Properties of the String Constructor</h1>
         <h1>String.fromCharCode ( ..._codeUnits_ )</h1>
         <p>The `String.fromCharCode` function may be called with any number of arguments which form the rest parameter _codeUnits_. The following steps are taken:</p>
         <emu-alg>
-          1. Let _codeUnits_ be a List containing the arguments passed to this function.
-          1. Let _length_ be the number of elements in _codeUnits_.
           1. Let _elements_ be a new empty List.
-          1. Let _nextIndex_ be 0.
-          1. Repeat, while _nextIndex_ &lt; _length_
-            1. Let _next_ be _codeUnits_[_nextIndex_].
+          1. For each element _next_ of _codeUnits_, in ascending index order, do
             1. Let _nextCU_ be ? ToUint16(_next_).
             1. Append _nextCU_ to the end of _elements_.
-            1. Increase _nextIndex_ by 1.
-          1. Return the String value whose code units are, in order, the elements in the List _elements_. If _length_ is 0, the empty string is returned.
+          1. Return the String value whose code units are, in order, the elements in the List _elements_. If the number of elements in _codeUnits_ is 0, the empty string is returned.
```

Comment:
can we use "If _codeUnits_ is empty, the empty string is returned", and maybe split this step up into an if/then two-step?

Reply:
I think it's useful to keep an explicit mention of the empty string case.

Reply:
You could make it a NOTE-step.

Reply:
A note seems like it’d apply to the whole algorithm, instead of just this step.

Reply:
A NOTE-*step* applies just to the point in the algorithm at which it occurs. (Scan the spec for `NOTE:`.)